### ISSUE TITLE
Clarify downtime start and end attributes

### ISSUE THREAD

**Issue by Apollorion** [DataDog/terraform-provider-datadog#1273]
### Terraform Version
1.0.10

### Affected Resource(s)
- datadog_downtime

In the docs within both github and the terraform registry [here](https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/downtime), the `start` and `end` attributes as documented are a tad confusing.

The `start` attribute for example is documented as `start (Number) Specify when this downtime should start`. I assumed the value here is an epoch timestamp, but I dont know what timezone to use. I assume UTC, but the documented example appears to be EST.

According to the the example:
```hcl
# Example: downtime for a specific monitor
# Create a new daily 1700-0900 Datadog downtime for a specific monitor id
resource "datadog_downtime" "foo" {
  scope      = ["*"]
  start      = 1483308000
  end        = 1483365600
...
```

Focusing on the `start` attribute purely (but this is also effecting the `end` attribute):
- `1483308000` in UTC is `Sunday January 01, 2017 22:00:00 (pm)`, which aligns with most conventions accounting for times in an API but not the example.
- `1483308000` in EST is `Sunday January 01, 2017 17:00:00 (pm)` which is  what the example says, but is outside convention.

So my questions here, is `start`/`end` expecting an epoch timestamp in UTC or EST?
Should I provide `1483290000` which is 1700 UTC or `1483308000` which is 1700 EST?

**Comment by Apollorion** [comment]
Looking further into this, only adds a bit more confusion. It looks like theres a `timezone` attribute that appears to default to `UTC`, but thats not being used in the example. So maybe the example is just wrong?

**Comment by ZsoltPath** [comment]
Further than that, TF is really bad in handling timestamp in number (epoch) format.
It'd be much more useful to be an RFC3339 string. Like start_date and end_date parameters are already.

**Comment by orlandothoeny** [comment]
Yes, it's really not intuitive. Resorted to creating a downtime manually in the UI, and checking the browser console to see which values are correct.

**Comment by ecdatadog** [comment]
Hello ! 

As of 2025, the recommended resource to handle monitor downtime is [datadog_downtime_schedule](https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/downtime_schedule). 
If you still use the `datadog_downtime`, it is important to note : 
* If you want one time downtime, you should fill : 
  * either `start` and `end` in epoch format (seconds since Jan 1 1970 UTC). Epoch format is always in UTC 
  * either `start_date` and `end_date` in ISO-8601 with UTC offset (+/-XX:YY or Z) 
  * `timezone` field will have no impact
* if you want recurrent downtime, you should fill : 
  * `recurrence` and `timezone` fields . 

For recurrent events, I recommend to look at this [examples pages](https://docs.datadoghq.com/monitors/downtimes/examples/?tab=api#downtime-outside-of-business-hours)
